images/packer/packer.json

```diff
@@ -0,0 +1,103 @@
+{
+  "variables": {
+    "aws_access_key": "",
+    "aws_secret_key": "",
+    "build_version": null,
```

Comment:
I wonder if we should just nix build_version completely here and use `isotime` in it's place... @chuckha thoughts?

Reply:
I'm +1 on this